CONTRIBUTING.md

```diff
@@ -1,32 +1,228 @@
-# Contributing
+# What The Hack - Contribution Guide
 
-The What The Hack project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit <https://cla.microsoft.com>.
+What The Hack is all about being "for the people, by the people". This repo was originally created to share real-world hackathons that Microsoft employees have hosted with their customers. Here are our core principles:
 
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+- Anyone can use the content to [host their own WTH event](./000-HowToHack/WTH-HowToHostAHack.md).
+- Anyone can [contribute a new hack](#-contribute-a-new-hack-to-what-the-hack).
+- Anyone can modify or update a hack as needed.
+  - Submitting a [pull request for updated content](#-contribute-an-update-to-an-existing-hack) is encouraged.
+- The content can always be shared with hack attendees **(AFTER your event)**
 
-# Code of Conduct
+This document provides the guidelines for how to contribute to the What The Hack project.
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+## How Can I Contribute?
 
-# Legal Notices
+The best way to contribute is to engage and get in touch with the What The Hack team by [submitting an issue via Github](https://github.com/microsoft/WhatTheHack/issues/new/choose). 
```

Comment:
Suggest using "contribute" or "engage", but not both.  

Or, you could say "The best way to contribute is to engage the WTH team by..."

I did note that there was nowhere to "request assistance" clearly.  Is that under "bug"?  If so, let's add that to the comment under "Bug" on submitting issue link with something along the lines of "Report a Hack Bug or Request Assistance".

Reply:
Done. Thanks for wordsmithing.

"Ask for help hosting a what The Hack" is how you request assistance.